openapi-to-har.js

```diff
@@ -104,21 +104,43 @@ const getPayload = function (openApi, path, method) {
   }
 
   if (
-    openApi.paths[path][method].requestBody &&
-    openApi.paths[path][method].requestBody.content &&
-    openApi.paths[path][method].requestBody.content['application/json'] &&
-    openApi.paths[path][method].requestBody.content['application/json'].schema
-  ) {
-    const sample = OpenAPISampler.sample(
-      openApi.paths[path][method].requestBody.content['application/json']
-        .schema,
-      { skipReadOnly: true },
-      openApi
-    );
-    return {
-      mimeType: 'application/json',
-      text: JSON.stringify(sample),
-    };
+     openApi.paths[path][method].requestBody &&
+     openApi.paths[path][method].requestBody.content
+   ) {
+     if (openApi.paths[path][method].requestBody.content['application/json'] &&
+       openApi.paths[path][method].requestBody.content['application/json'].schema
+     ) {
+       const sample = OpenAPISampler.sample(
+         openApi.paths[path][method].requestBody.content['application/json']
+           .schema,
+         { skipReadOnly: true },
+         openApi
+       );
+       return {
+         mimeType: 'application/json',
+         text: JSON.stringify(sample)
+       };
+     }
+
+     if (openApi.paths[path][method].requestBody.content['multipart/form-data'] &&
+       openApi.paths[path][method].requestBody.content['multipart/form-data'].schema
+     ) {
+       const sample = OpenAPISampler.sample(openApi.paths[path][method].requestBody.content['multipart/form-data']
+           .schema,
+         {skipReadOnly: true},
+         openApi
+       );
+
+       if (sample === undefined) return null;
+
+       const params = [];
+        Object.keys(sample).forEach(key => params.push({'name': key, 'value': sample[key]}));
```

Comment:
Unexpected indentation
```suggestion
      Object.keys(sample).forEach(key => params.push({'name': key, 'value': sample[key]}));
```